cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -174,6 +177,13 @@ private void setDefaultRegulatingControl(CgmesRegulatingControlForStaticVarCompe
         }
     }
 
+    // SVCControlMode and voltageSetPoint are optional in Cgmes 3.0
+    private static boolean defaultRegulatingControlIsWellDefined(CgmesRegulatingControlForStaticVarCompensator rc) {
+        return rc.defaultRegulationMode != null
+                && (RegulatingControlMapping.isControlModeVoltage(rc.defaultRegulationMode.toLowerCase()) && Double.isFinite(rc.defaultTargetVoltage) && rc.defaultTargetVoltage > 0.0
```

Comment:
No need to check `defaultTargetVoltage` is finite, it is enough to check it is `> 0`

Reply:
Done